Makefile

```diff
@@ -75,32 +99,32 @@ docs-user: clean-docs
 docs: docs-user docs-developer
 
 release:
-	ls -l dist/
-	read "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."
+	@ls -l dist/
+	@echo "\nDo you want to upload everything in dist/*?\n\n CTRL+C to exit."
+	@read __
 	twine upload -s dist/*
 
 staticdeps:
 	rm -r kolibri/dist/* || true # remove everything
 	git checkout -- kolibri/dist # restore __init__.py
-	pip install -t kolibri/dist -r $(REQUIREMENTS)
-	python install_cexts.py --file $(REQUIREMENTS_CEXT) # pip install c extensions
-	pip install -t kolibri/dist -r $(REQUIREMENTS_CEXT_NOARCH) --no-deps
+	pip install -t kolibri/dist -r "requirements.txt"
+	python install_cexts.py --file "requirements/cext.txt" # pip install c extensions
+	pip install -t kolibri/dist -r "requirements/cext_noarch.txt" --no-deps
 	rm -r kolibri/dist/*.dist-info  # pip installs from PyPI will complain if we have more than one dist-info directory.
 
 writeversion:
 	python -c "import kolibri; print(kolibri.__version__)" > kolibri/VERSION
+	@echo ""
+	@echo "Current version is now `cat kolibri/VERSION`"
 
-setrequirements:
```

Comment:
Why is this removed?

Reply:
It's all in the `buildconfig` target now... sort of a normalization, but also a bug fix... see: #3129

Reply:
No wait, it's not a bug fix... just merging to Make targets that always got applied sequentially.

Reply:
This line `python build_tools/customize_requirements.py` has been omitted and is not reinstated anywhere else as far as I can see? This will make our custom builds impossible.

Reply:
Ah yes, is reinstated now!